src/main/java/de/berlin/lostberlin/service/mail/MailSender.java

```diff
@@ -21,7 +21,7 @@
 	@Autowired
 	SendGrid sendGrid;
 	
-	@Value("mail.from")
+	@Value("${mail.from}")
 	private String from;
```

Comment:
why do you need it here?

Reply:
Sendgrid doesn't recognise it otherwise,  I get IO exception when trying to send a mail. You wired other two templates in the same way.

Reply:
Question is why do you wire the value here when it is already wired in MailService? A MailSender should take all parameters from the Params object, not by wiring them from the outside on its own.

Reply:
oh ok, thanks for the explanation. I didn't think of it at first, because this wiring was in your commit already. Already fixed it:)